eclass/qt5-build.eclass

```diff
@@ -720,11 +737,37 @@ qt5_tools_configure() {
 	# allow the ebuild to override what we set here
 	myqmakeargs=( "${qmakeargs[@]}" "${myqmakeargs[@]}" )
 
+	# out-of-source build quirks
+	case ${PN} in
+		qdoc)
+			# bug 676948
+			qt5_tools_oos_quirk src/qdoc qtqdoc-config.pri
+			;;
+		qtlocation)
+			# src/plugins/geoservices requires files that are only generated
+			# when qmake is run in the root directory. bug 633776
+			qt5_tools_oos_quirk src/location qtlocation-config.pri
+			;;
+		*) ;;
+	esac
+
 	mkdir -p "${QT5_BUILD_DIR}" || die
 	qt5_qmake "${QT5_BUILD_DIR}"
 	cp qttools-config.pri "${QT5_BUILD_DIR}" || die
 }
 
+# @FUNCTION: qt5_tools_oos_quirk
+# @INTERNAL
+# @USAGE: <path> <file>
+# @DESCRIPTION:
+# Runs qmake in root directory, copies generated pri <file> from source <path>
```

Comment:
Sorry to be annoying, but oos may not be obvious to everyone (I know chances of others reading it are small anyway, but well):
```suggestion
# Quirk for out-of-source builds. Runs qmake in root directory, copies generated pri <file> from source <path>
```

Reply:
That's what I'm here for.